docs/features.rst

```diff
@@ -17,11 +17,14 @@ Storing features
 ****************
 
 Features in Lhotse are stored as numpy matrices with shape ``(num_frames, num_features)``.
-By default, we use `lilcom`_ for lossy compression and reduce the size on the disk approximately by half.
+By default, we use `lilcom`_ for lossy compression and reduce the size on the disk by about 3x.
 The lilcom compression method uses a fixed precision that doesn't depend on the magnitude of the thing being compressed, so it's better suited to log-energy features than energy features.
-For now, we extract the features for the whole recordings, and store them in separate files.
-We retrieve them by loading the whole feature matrix and selecting the relevant region (e.g. specified by a cut).
-Eventually we will look into optimizing the storage to further reduce the I/O.
+We currently support two kinds of storage:
+
+- HDF5 files with multiple feature matrices (recommended)
+- directory with feature matrix per file (not recommended, costly I/O)
+
+We retrieve the arrays by loading the whole feature matrix from disk and selecting the relevant region (e.g. specified by a cut). Therefore it makes sense to cut the recordings first, and then extract the features for them to avoid loading unnecessary data from disk (especially for very long recordings).
```

Comment:
We might at some point need to implement a storage method that avoids this decision... e.g. use lilcom to separately store features for 1 second at a time and figure out a transparent way to load this.

Reply:
Sounds good. It's probably good to drive the decisions where to optimize based on the training speed/bottlenecks when we have some recipes.